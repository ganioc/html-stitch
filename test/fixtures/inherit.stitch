{{ inherit _basic.stitch }}

{{ include _laugh.stitch:_cry.stitch }}
