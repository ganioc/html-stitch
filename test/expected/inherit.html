<html>
<head>
<i>Crying face!</i>
</head>
<body>

</body>
</html>
